src/test/TxTests.cpp

```diff
@@ -180,7 +181,7 @@ void
 checkTransaction(TransactionFrame& txFrame, Application& app)
 {
     REQUIRE(txFrame.getResult().feeCharged ==
-            app.getLedgerManager().getTxFee());
+            getCurrentTxFee(app.getLedgerStateRoot()));
```

Comment:
I think you can as well rename it to getCurrentOpFee, as in reality it is fee per operation, not per transaction